include/SDL3/SDL_video.h

```diff
@@ -1363,7 +1416,17 @@ extern DECLSPEC int SDLCALL SDL_HideWindow(SDL_Window *window);
 extern DECLSPEC int SDLCALL SDL_RaiseWindow(SDL_Window *window);
 
 /**
- * Make a window as large as possible.
+ * Request that the window be made as large as possible.
+ *
+ * You can't maximize a non-resizable window.
```

Comment:
Does this fail, or is it deferred until the next opportunity?

Reply:
Trying to maximize a window without the resizable attribute fails. Every backend already disallowed it, this just codifies it.

Will improve the documentation.